content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,67 @@
+
+This was the same intuition we used while initially designing Delphi's data monitoring system at the pandemic's start. This monitoring system was particularly important because Delphi's stakeholders wanted to be aware of any events in the data - like changes in data quality or disease dynamics. First, we split these events into data validation (e.g., negative cases) and events that needed a human-facing statistical alert (using the above formula) so that an expert could ultimately interpret and classify the event. 
+
+However, we quickly realized this formula did not work with Delphi's large data volume. To illustrate, creating alerts for 1% of the data over millions of data points, like those analyzed by Delphi, results in 10,000+ alerts - far too many for us at Delphi to inspect and investigate! Standard correction approaches were also incompatible with public health data. Our first attempt was reducing the percentage of alerts (e.g., thresholds that would identify from 1% of the data as alerts to 0.01%). However, because modeling public health data is difficult, defining thresholds (e.g., 1 in 10000) when data streams only have a few hundred historical data points can be somewhat arbitrary. Additionally, when applying the same standard threshold-generating function across all the data (adaptive thresholds), we missed some important events from signals that were not well-calibrated to the function. Tuning these parameters by hand across hundreds of signals was tedious, and Delphi turned off the outlier alerting system. 
+
+The historical alerting framework cannot keep up with monitoring the increasingly large volumes of heterogeneous public health data that organizations like Delphi process. Delphi has increased its volume by 1000x in the past three years alone. And, although Delphi processes data from several geographical regions, even local public health agencies who use systems (that focus on one geography) under the existing alerting framework have challenged its usefulness (e.g. "[What Can You Really Do with 35,000 Statistical Alerts a Week Anyways?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6606136/)" (2019)). We anticipate this failure mode for alerting systems will become more prevalent as data volumes grow. 
+
+### Bridging the gap with FlaSH:
+
+Identifying and communicating noteworthy events from public health data is a critical step for future pandemic prevention <sup>[4](#link4)</sup>. But how should we redesign our event detection process? 
```

Comment:
This superscript does not suffer the same spacing strangeness as in the first paragraph.  Very weird!